FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,123 @@
+# Currently lightmap baking cannot be performed at runtime, so it's not terribly useful yet.
+# Consider creating a procedural Voxel GI solid class entity that can be baked at runtime instead.
+func _build_complete() -> void:
+    print("Success! Unwrapping UV2...")
+    func_godot_map.unwrap_uv2()
+
+func _build_failed() -> void:
+    printerr("Failed to build the map file! :(")
+
+func _unwrap_uv2_complete() -> void:
+    print("UV2 unwrapping completed!")</code></pre>
+    <p>An example of this can be found in <a href="https://github.com/func-godot/func_godot_runtime_building" target="_blank">func_godot_runtime_building repository</a>.</p>
+
+    <h1>Building from Map Editors</h1>
+    <p>
+        Much like idTech engine games, it's possible to build and launch directly into a map from some map editors.
+        This can be quite a helpful tool for testing rapid iterations.
+    </p>
+
+    <h2>Godot Project Setup Example</h2>
+    <p>To build and launch our maps from the editor, you'll need to employ the <a href="https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-method-get-cmdline-user-args" target="_blank">OS class' ability to retrieve command line arguments</a>. A basic example might look something like this:</p>
+
+
+    <code><pre># This script should be autoloaded to handle command line arguments
+# Additionally, requires a valid reference to a FuncGodot map
+
+var func_godot_map: FuncGodotMap
+
+var map_override: String
+var cmd_line_args := OS.get_cmdline_user_args()
+
+# Loop over all command line arguments, split on a delimiter for each command, and use this for the file path
+for cmd_line_arg in cmd_line_args:
+    if cmd_line_arg.contains("run_map::"):
+        var arg_and_value := cmd_line_arg.split("::")
+        # This should split into an array of size 2, containing the parameter name and its corresponding value
+        if arg_and_value.size() != 2:
+            return
+
+        map_override = arg_and_value[1]
+
+        func_godot_map.local_map_file = map_override
+        func_godot_map.verify_and_build()</code></pre>
+    <h2>Setting Up Your Editor</h2>
+    <p>In the below guides, there will be references made to executables. This is up to the user, however you can use the Godot executable to run your game straight from the editor, or you could use exported binaries of your game. On Windows, your Godot Binary path is where you keep your Godot.exe file. On most Linux systems this is found under <code>/usr/bin/godot</code>. An example executable path and parameters for running a game in the editor: <code class="highlight">C:/GAMEDEV/Godot.exe --path C:/func_godot_project_example/ -- run_map::${MAP_BASE_NAME}.map</code>. Note that <code class="highlight">--path</code> is to the directory where project.godot resides. See the full list of Godot engine <a href="https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html" target="_blank">Command Line Interface options</a>.</p>
```

Comment:
Reads a little weird. Rewriting for succinctness and clarity. Also, I would hope a Linux user knows where to find the Godot Editor binary. Removing the example parameter

```html
<p>In every map editor you'll need to specify both an <b>executable</b> and <b>parameters</b> in your launch or build options. The executable can be either the <b>exported binary of your game</b> or the <b>Godot Editor binary</b> if launching from a Godot project. To run from your game from the Godot project, be sure to use the <code>--path</code> parameter, as defined in <a href="https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html" target="_blank"><b>Godot's Command Line Interface options</b></a>.</p>
```
Reads more cleanly, and we'll worry about proper parameter syntax and examples on a per engine basis.